src/components/EventList.spec.tsx

```diff
@@ -34,17 +34,32 @@ describe('EventList', () => {
     });
 
     it('contains an activity title instead of missing event title', () => {
-        dummyEvents.data[0].title = undefined;
-        mount(<EventList events={ dummyEvents.data } org={ dummyOrg.data }/>);
+        dummyEvents[0].title = undefined;
+        mount(<EventList events={ dummyEvents } org={ dummyOrg }/>);
 
         cy.get('[data-test="event"]')
-            .should('contain', dummyEvents.data[0].activity.title)
+            .should('contain', dummyEvents[0].activity.title)
             .should('not.contain', 'undefined');
     });
 
     it('contains a sign-up button for each event', () => {
-        mount(<EventList events={ dummyEvents.data } org={ dummyOrg.data }/>);
+        mount(<EventList events={ dummyEvents } org={ dummyOrg }/>);
 
         cy.get('[data-test="sign-up-button"]').should('be.visible');
     });
+
+    it('shows a placeholder when the list is empty', () => {
+        dummyEvents = [];
+        mount(<EventList events={ dummyEvents } org={ dummyOrg }/>);
+
+        cy.get('[data-test="no-events-placeholder"]').should('be.visible');
+    });
+
+    it('shows a placeholder when the list is undefined', () => {
+        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+        dummyEvents = undefined!;
+        mount(<EventList events={ dummyEvents } org={ dummyOrg }/>);
```

Comment:
You don't need to store `undefined` in a variable. Just pass it straight into the component using `events={ undefined }`.